Card Focus/CardFocus.js

```diff
@@ -70,25 +75,36 @@ tau.mashups
             },
 
             focusOnCards: function() {
-                var clipboardManager = configurator.getClipboardManager();
+                var clipboardManager = appConfigurator.getClipboardManager();
 
                 var cards = _.values(clipboardManager._cache);
 
-                var ids = cards.reduce(function(r, item) {
-                    r.push(item.data.id); return r;
-                }, []);
+                var ids = _.chain(cards)
```

Comment:
I suppose you can replace this 8 lines with 3 lines of `reduce` and rid of obscure underscore API.